whirlpool/appliance.py

```diff
@@ -15,6 +15,7 @@
 
 SETVAL_VALUE_OFF = "0"
 SETVAL_VALUE_ON = "1"
+RETRY_COUNT = 3
```

Comment:
```suggestion
REQUEST_RETRY_COUNT = 3
```
Just to make it a bit more explicit